tests/apk-sizes-reference/Xamarin.Forms_Performance_Integration-Signed-Release-Bundle.apkdesc

```diff
@@ -5,7 +5,7 @@
       "Size": 3768
     },
     "classes.dex": {
-      "Size": 2802888
+      "Size": 2976868
```

Comment:
So is this why we didn't set `$(AndroidProguardMappingFile)` by default before? Because it regresses .apk size? This might be fine?

If someone didn't want this mapping, it doesn't look like there is a way to turn if off now?

Reply:
It might well be the reason why we didn't set it. 
So perhaps we should leave it off by default still.

Reply:
For developers submitting to Google Play, it seems like they would probably rather have a 17kb larger `classes.dex` file and the warning in Google Play would go away?

Maybe we should just add a new boolean property to turn this on or off, and default to on?

Reply:
Maybe this might work?

```
<Target Name="_SetProguardMappingFileProperty">
	<PropertyGroup>
		<AndroidProguardMappingFile Condition=" '$(AndroidLinkTool)' == 'r8' And '$(AndroidProguardMappingFile)' != 'None' ">$(OutputPath)mapping.txt</AndroidProguardMappingFile>
	</PropertyGroup>
</Target>
```

Then if a user wants to opt-out then can define 

```
< AndroidProguardMappingFile>None</AndroidProguardMappingFile>
```

or 

```
-p: AndroidProguardMappingFile=None
```

Reply:
Seems like that would work? Just couldn't name the file `None`, lol.